packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -86,10 +84,11 @@ RadioGroup.displayName = RADIO_GROUP_NAME;
 const ITEM_NAME = 'RadioGroupItem';
 
 type RadioGroupItemElement = React.ElementRef<typeof Radio>;
-type RadioGroupItemProps = Radix.MergeProps<
-  Omit<Radix.ComponentPropsWithoutRef<typeof Radio>, 'onCheck'>,
-  { value: string; name?: never }
->;
+type RadioProps = Radix.ComponentPropsWithoutRef<typeof Radio>;
+interface RadioGroupItemProps extends Omit<RadioProps, 'onCheck'> {
+  value: string;
+  name?: never;
+}
```

Comment:
do we still need to do this now, or could we get away with doing:

```suggestion
interface RadioGroupItemProps extends Omit<RadioProps, 'onCheck' | 'name'> {
  value: string;
```

Reply:
Good catch! 👀!!